### PR TITLE
Added Layer5 repositories

### PR DESCRIPTION
**Pull request by iambami**
Signed-off-by: Atinuke Bami Kayode <titatinuks@gmail.com>

#### ℹ️ Repository information
https://github.com/layer5io/layer5/ 
https://github.com/meshery/meshery

**The repository has**: Good first issues.

- [x] At least three issues with the good first issue label.
- [x] Detailed setup instructions for the project.
- [x] CONTRIBUTING.md
- [x] Actively maintained.

### CODE REVIEW

**Comment by iambami** [comment]
Hi @brylie I created a PR two days ago. Can you review it, please.

File: .deepsource.toml [hunk 1]
```diff
@@ -34,3 +34,5 @@ enabled = true
 [[analyzers]]
 name = "secrets"
 enabled = true
+
+
```
Comment (sanket-deepsource) [inline .deepsource.toml:38]: Can you please remove these additional newlines?

File: data/repositories.toml [hunk 2]
```diff
@@ -370,6 +370,8 @@ repositories = [
   'github.com/kubernetes/minikube',
   'github.com/kubernetes/test-infra',
   'github.com/kubernetes/website',
+  'github.com/layer5io/layer5',
+  'github.com/meshery/meshery',
```
Comment (sanket-deepsource) [inline data/repositories.toml:374]: Please add this in the correct lexicographic order.

**Review by sanket-deepsource — CHANGES_REQUESTED** [review]

File: .deepsource.toml (hunk 1 above)
Comment (iambami) [reply .deepsource.toml:38]: I will see to it. Thank you

**Comment by iambami** [comment]
Requested changes have been made. Thank you.

File: .deepsource.toml (hunk 1 above)
Comment (siddhant-deepsource) [inline .deepsource.toml:38]:
```suggestion
```

**Review by siddhant-deepsource — COMMENTED** [review]
Thanks @iambami 🙌 Great projects.

**Review by siddhant-deepsource — APPROVED** [review]

**Comment by iambami** [comment]
Thanks @sanket-deepsource 
Looking forward to have more folks contributing to open source through Layer5.

**Review by sanket-deepsource — APPROVED** [review]

**Pull request merged**